Stripe/PublicHeaders/STPPaymentHandler.h

```diff
@@ -87,7 +87,7 @@ typedef NS_ENUM(NSInteger, STPPaymentHandlerErrorCode) {
      If you're using Apple Pay, you must implement `STPAuthenticationContext prepareAuthenticationContextForPresentation:`
      */
     STPPaymentHandlerRequiresAuthenticationContextErrorCode,
-};
+} NS_SWIFT_NAME(STPPaymentHandler.ErrorCode);
```

Comment:
The Swift bridge is parsing these out as `STPPaymentHandler.ErrorCode.requiresPaymentMethodErrorCode`, so you might want to add something like `STPPaymentHandlerRequiresPaymentMethodErrorCode NS_SWIFT_NAME(requiresPaymentMethod),` to the values.